tests/store/model_registry/test_rest_store.py

```diff
@@ -200,14 +203,23 @@ def test_delete_registered_model_tag(self, mock_http):
 
     @mock.patch("mlflow.utils.rest_utils.http_request")
     def test_create_model_version(self, mock_http):
+        run_id = uuid.uuid4().hex
```

Comment:
Could this in theory just be `"some-run-id"` because the test stuff gets cleared anyways?

Reply:
mm yeah i think so. plus run IDs can be duplicative for model versions.